src/test/netbase_tests.cpp

```diff
@@ -65,6 +65,7 @@ BOOST_AUTO_TEST_CASE(netbase_properties)
     BOOST_CHECK(ResolveIP("2001:10::").IsRFC4843());
     BOOST_CHECK(ResolveIP("2001:20::").IsRFC7343());
     BOOST_CHECK(ResolveIP("FE80::").IsRFC4862());
+    BOOST_CHECK(ResolveIP("FEC0::").IsRFC3897());
```

Comment:
Could add `BOOST_CHECK(ResolveIP("FEC0::").IsRoutable());` and `BOOST_CHECK(!ResolveIP("FEB0::").IsRoutable());` to cover one positive and one negative case :)

Reply:
Sure, added ✌️